x-pack/plugins/lists/server/routes/delete_exception_list_route.ts

```diff
@@ -39,7 +39,6 @@ export const deleteExceptionListRoute = (router: IRouter): void => {
             statusCode: 400,
           });
         } else {
-          // TODO: At the moment this will delete the list but we need to delete all the list items before deleting the list
```

Comment:
Yep, that is an incorrect TODO that shouldn't have been there. When you delete an exception_list it will automatically delete all the list items through deleteFoundExceptionListItems here:

```ts
kibana/x-pack/plugins/lists/server/services/exception_lists/delete_exception_list_items_by_list.ts
```

It's slow though since we don't have bulk delete or delete by query from saved objects at this moment but I still remove the list items.